klpbuild/config.py

```diff
@@ -80,8 +80,10 @@ def setup_user_env(self, basedir):
         datadir = Path(basedir, "data")
 
         config = configparser.ConfigParser(allow_no_value=True)
+
         config['Paths'] = {'work_dir': workdir,
-                           'data_dir': datadir}
+                           'data_dir': datadir,
+                           '#kernel_src_dir': 'kernel-src/ (e.g. https://github.com/SUSE/kernel-source)'}
```

Comment:
I think that kernel-source doesn't contain the CVE branches that we have on kerncvs, so we may remove it. Also, IIRC, current we don't require this value to be defined, so I think that we can relax the check for this setting.

Per my tests it fails if kernel_src_dir is not set, or if it's created by klpbuild but it's commented.